luminoth/tools/dataset/readers/object_detection/object_detection_reader.py

```diff
@@ -0,0 +1,22 @@
+import abc
+
+from luminoth.tools.dataset.readers import BaseReader
+
+
+class ObjectDetectionReader(BaseReader):
+    """Reads data suitable for object detection.
+
+    Object detections needs:
+        - images
+        - gt_boxes with labels
+    """
+    def __init__(self):
```

Comment:
Perhaps this docstring should be changed to mention what iterate() needs to return.

```
dict with
  int width
  int height
  int depth
  str filename
  bytes image_raw
  list of dicts gt_boxes:
    'label': int
    'xmin': int?
    'ymin': int?
    'xmax': int?
    'ymax': int?
```